ui/app/templates/components/shamir-flow.hbs

```diff
@@ -4,11 +4,7 @@
       <HoverCopyButton @copyValue={{encoded_token}} />
       <div class="message-body">
         <h4 class="title is-7 is-marginless">
-          {{#if otp}}
-            Encoded Operation Token
-          {{else}}
-            Encrypted Operation Token
-          {{/if}}
+          Encoded Operation Token
```

Comment:
Is there a difference here between encoded and encrypted?  Is it important to keep the word encrypted on the refresh situation?

Reply:
thanks for this question! i wondered the same thing. all of the examples in this previous commit [[1](https://github.com/hashicorp/vault/pull/5818/files#diff-115a2172c354bb1dbf2988a9fd7f099eL120-L122)][1](https://github.com/hashicorp/vault/pull/5818/files#diff-115a2172c354bb1dbf2988a9fd7f099eL13)] made it sound like the resulting value is actually encoded -- i couldn't figure out why we ever referred to it as encrypted. @meirish do you know?

Reply:
I think originally this was meant to denote the difference between the PGP encrypted, and the OTP which does an XOR (https://en.wikipedia.org/wiki/XOR_cipher#Use_and_security) to decode the token, but I don't think it's important to keep. Looking at the wikipedia page, OTP stands for "one time pad" not "one time password" which I didn't realize when building this the first time.